packages/app/src/App.vue

```diff
@@ -23,7 +23,7 @@ export default defineComponent({
         // if not authenticated and on page that requests authentication
         const pageAuthentication = route.meta.authentication || 'needed';
         if (pageAuthentication === 'needed' && !_isAuthenticated) {
-          router.replace({ name: 'login' });
+          router.replace({ name: 'welcome' });
```

Comment:
This now means: If you go to (or if you are still on) a page which requires authentication and you are not authenticated, redirect the user to welcome. 

As a user I would still expect to land on the login page in such a case. The welcome screen should come each time I open the app without requesting a specific page (like opening the first time with the base url).

Reply:
In our current design, there is no other page with the option to choose your login method. What should be displayed on an 'login' screen, which is not already on the welcome screen?

Reply:
I agree with that. So can we delete the old login page and all links / references to it then?

Reply:
I would suggest that.